core/App/hooks/notifications.ts

```diff
@@ -14,8 +16,12 @@ interface Notifications {
 export const useNotifications = (): Notifications => {
   const offers = useCredentialByState(CredentialState.OfferReceived)
   const proofs = useProofByState(ProofState.RequestReceived)
+  const revoked = useCredentialByState(CredentialState.Done).filter((cred: CredentialRecord) => {
```

Comment:
I think this would be good long term but we would need to have a conversation with the AFJ community about how they want to do this. Using a metadata property on the credential to know if the notification has been seen is not an approach we've discussed yet